prow/config/config.yaml

```diff
@@ -529,6 +529,48 @@ branch-protection:
                   - 'idc-jenkins-ci/leak-test'
                   - 'license/cla'
                 strict: true
+        tidb-binlog:
+          branches:
+            master:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'idc-jenkins-ci-binlog/build'
+                  - 'idc-jenkins-ci-binlog/check'
+                  - 'idc-jenkins-ci-tidb-binlog/integration-test'
+                  - 'idc-jenkins-ci-tidb-binlog/check_release_note'
+                  - 'idc-jenkins-ci-tidb-binlog/unit-test'
+                strict: true
```

Comment:
If we don't use the tars, I think it should be false.

Reply:
What is the reason for setting to false?

Reply:
This is whether to request merge base.